### PR TITLE
DOCS-321: Add Replication section to TrueNAS docs and begin building content

### PR DESCRIPTION
**Pull request by Mrt134**
Restructure to add a Replication category to the TrueNAS docs.
Add placeholders for additional articles
Write an Initial howto for quick replications, with theme-agnostic images.
hugo local testing: no issues.

Thanks for contributing to FreeNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by Mrt134** [comment]
Added another prerequisite article about configuring SSH to this PR. This article was required before writing the Remote replications articles.

File: content/en/docs/TrueNAS/ssh-connections.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+Don't forget to re-enable the SSH service on the **Services** page when all the configuration changes are complete.
+
+## Restricting Command Line Users to *scp* or *sftp*
+
+This only works for users that use command line versions of *scp* and *sftp*.
+When SSH is configured, authenticated users with a user account can use *ssh* to log into the TrueNAS system over the network.
+User accounts are created by going to **Accounts > Users** and clicking ADD.
+While the SSH login defaults to the user home directory, users are able to navigate outside their home directory, which can pose a security risk.
+
+To allow users to use *scp* and *sftp* to transfer files between their local computer and their home directory on the TrueNAS system, while restricting them from logging into the system using *ssh*.
```
Comment (a-stjohn) [inline content/en/docs/TrueNAS/ssh-connections.md]: This sentence might be missing something? or contain an extra word? or punctuation?

File: content/en/docs/TrueNAS/ssh-connections.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+### Adding a Public SSH Key to the TrueNAS Root Account
+
+Log in to the TrueNAS system that generated the SSH keypair and go to **System > SSH Keypairs**.
+Open the keypair to use for the SSH connection and copy the text of the public SSH key or download the public key as a text file.
+
+<img src="/images/system-sshkeypairs-copypublic.png">
+<br><br>
+
+Log in to the TrueNAS system and go to **Accounts > Users**.
```
Comment (a-stjohn) [inline content/en/docs/TrueNAS/ssh-connections.md]: May need to clarify which TrueNAS system here. I'm assuming it's the TrueNAS that didn't generate the SSH keypari?

File: content/en/docs/TrueNAS/automated-tasks/replication/legacy.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+title: "Legacy Engine (11.3)"
+linkTitle: "Legacy Engine (11.3)"
+description: "How to create a replication task using the previous FreeNAS/TrueNAS 11.2 replication engine"
+weight: 4
+---
+
+{{% alert title="Notice" color="info" %}}
+This article only applies to FreeNAS or TrueNAS version 11.3.
+The "Legacy" option is available in this version of FreeNAS or TrueNAS to provide compatability with the replication engine integrated in FreeNAS/TrueNAS 11.2 and earlier.
+{{% /alert %}}
```
[the remark this answers is not in the document]
Comment (Mrt134) [reply content/en/docs/TrueNAS/automated-tasks/replication/legacy.md]: The shortcode doesn't quite fit for this article as it is very specific about the version that it applies to. I have gone back and added the versioning to the other articles though, so thanks for the link.

**Pull request merged**